Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,39 @@
-Add your answers to the Algorithms exercises here.
```

Comment:
Kyle your answers show you really dived into Big-O as a topic 👍 Good job breaking down each code snippet in detail, and sharing your thought process.